ex2.c

```diff
@@ -29,7 +29,6 @@
 #include <time.h>
 
 #include <paraconf.h>
-// load the PDI header
 #include <pdi.h>
```

Comment:
May keep the comment "// load the PDI header".

Reply:
will be fiexd by the this issue of pdidev/pdi  Add full configuration example to Trace plugin #499